build_test.go

```diff
@@ -58,6 +50,16 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 
 		buildpackYMLParser = &fakes.BuildpackConfigParser{}
 
+		portChooserLayer = packit.Layer{
+			Path:             "<UNKNOWN>",
```

Comment:
This can be:
` Path:             filepath.Join(layersDir, "port-chooser")`

and then wherever you are doing `portChooserLayer.Path = result.Layers[0].Path` can be removed.

Reply:
Done, good idea

Reply:
done, will push in a minute